drv/lpc55-gpio/src/main.rs

```diff
@@ -51,6 +51,19 @@ task_slot!(SYSCON, syscon_driver);
 
 struct ServerImpl<'a> {
     gpio: &'a device::gpio::RegisterBlock,
+    pint: &'a device::pint::RegisterBlock,
+    inputmux: &'a device::inputmux::RegisterBlock,
+}
+
+impl ServerImpl<'_> {
+    fn set_pin_direction(&self, port: usize, pin: usize, dir: Direction) {
+        match dir {
+            Direction::Input => self.gpio.dirclr[port]
+                .write(|w| unsafe { w.dirclrp().bits(1 << pin) }),
```

Comment:
**question:** Would adding safety comments while moving this code be worthwhile, or is it immediately obvious?

Reply:
The unsafe is a quirk of how the lpc55-pac is generated. Adding a safety comment for this is unlikely to be useful.